src/main/java/com/agnieszkapawska/flashcards/domain/models/Role.java

```diff
@@ -0,0 +1,52 @@
+package com.agnieszkapawska.flashcards.domain.models;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "role")
```

Comment:
Flashcard, QuestionTag but role, user... Maybe we could keep one convention for naming tables?